_includes/components/banner.html

```diff
@@ -16,6 +16,11 @@
       </div>
     </header>
     <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+      <script type="text/javascript">
+        // Hide the gov banner immediately rather than waiting for external JS
+        // sources to load and execute.
+        document.getElementById("gov-banner").setAttribute("hidden");
```

Comment:
The second argument is required, otherwise I'd expect this may error.

```
Uncaught TypeError: Failed to execute 'setAttribute' on 'Element': 2 arguments required, but only 1 present.
```

```suggestion
        document.getElementById("gov-banner").setAttribute("hidden", "");
```